context/app/static/js/pages/Error/style.js

```diff
@@ -15,8 +15,9 @@ const Background = styled.div`
 const StyledPaper = styled(Paper)`
   width: 100%;
   max-width: 880px;
-  margin: 0px 16px;
-  padding: 16px;
+  margin: 0 1rem;
+  padding: 1rem;
```

Comment:
Prefer to use spacing from the theme for now which uses `px`. We can consider using `rem` when we overhaul styling down the road.

Reply:
Makes sense; I've reverted this to use `px`.